pkg/mountinfo/mountinfo_test.go

```diff
@@ -0,0 +1,636 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Tests are failing because of missing build tags.
```
Test file(s) does not contain a tag privileged_tests or !privileged_tests tags
```
please add this line
```golang
// limitations under the License.

// +build !privileged_tests

package mountinfo
```